examples/ansible/code/Synchronize_DNS_Zones

```diff
@@ -0,0 +1,50 @@
+- name: Sync DNS
+  hosts: <your Bind-Server>
+  tasks:
+    - name: get Managed Zones Names
+      ansible.builtin.set_fact:
+        zone_names: "{{ query('netbox.netbox.nb_lookup', 'zones', plugin='netbox_dns',
+                           api_endpoint='https://<nextcloud-url>/',
+                           api_filter='status=active status=deprecated',
+                           token='<redacted>')
+                    | map(attribute='value') | map(attribute='name') }}"
+
+    - name: Generate Zone Files
+      template:
```

Comment:
Could you please use FQCN notation for all tasks? Over the last few years, FQCN has become mandatory for some linters and is considered correct code style - it's a tedious task to clean it up everywhere, especially in your own old code, but it's certainly worth it to start doing it for new code.